release/release-source

```diff
@@ -248,6 +250,8 @@ shift $(expr $OPTIND - 1)
 
 if [ $VERBOSE -eq 1 ]; then
     set -x
+elif [ $QUIET -eq 0 ]; then
+    MAKE="make --silent"
```

Comment:
This looks the wrong way around -- should't we use `--silent` if `$QUIET` is true, i. e. `-eq 1`, not 0?